backend/src/routes/v1/acronyms.js

```diff
@@ -31,4 +31,39 @@ acronyms.get('/:appAcronym', [
   }
 });
 
+// A special administrative call to add users to acronym. This is a temporary shim until we have an actual administrative
+// section of the app in place
+acronyms.get('/:appAcronym/addUser/:username', [
+], async (req, res) => {
+  // Check for required permissions
+  if (!req.user.jwt.realm_access.roles.includes('GETOK_ADMIN_ADD_USER')) {
+    return res.status(403).json({
+      message: 'Access Denied'
+    });
+  }
+
+  if (!req.params.appAcronym || !req.params.username) {
+    res.status(400).json({
+      message: 'Must supply app acronym and user (ex: myname@idir)'
+    });
+    return res;
+  }
+
+  try {
+    const token = req.headers.authorization.split(' ')[1];
+    const response = await acronymComponent.registerUserToAcronym(token, req.user.jwt.iss, req.params.appAcronym.toUpperCase(), req.params.username.toLowerCase());
```

Comment:
saw jujaga's comment about moving uppercase and lowercase into the registerUserToAcronym function - would be a good idea.

Reply:
Fixed